### PR TITLE
added insert and remove functions for both concerts and artists tables as well as testing for these functions

### PR DESCRIPTION
**Pull request by EricSmith10**

### CODE REVIEW

**Comment by eremozdemir** [comment]
Tests all run and views is at a good spot!

**Comment by KevinKim99** [comment]
clean coded, looks good, tests run in a perfect way, good work!

File: Database/mysite/myapp/views.py
Comment (KenichiroAnderson) [inline Database/mysite/myapp/views.py:1]: looks good, good idea on the try catch statements!.

File: Database/mysite/myapp/tests.py
Comment (KenichiroAnderson) [inline Database/mysite/myapp/tests.py:1]: Nice testing implementation, also good on the commenting helps make these files readable

**Review by KenichiroAnderson — COMMENTED** [review]
I like the try catch statements. this should help in debugging. also thank you for commenting on these files, helps make them readable!

**Comment by NathanIsCoding** [comment]
Everything looks good to me, nicely laid out good work!

**Pull request merged**